src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: 96px;
+  justify-content: space-between;
+  padding-right: 4px;
```

Comment:
you really don't need this padding, better give `margin-right` for each star except the last one in `stars__star` block. for ex. you can go something like this:
```
.stars__star:not(:last-child) {
  margin-right: 4px;
}
```

Reply:
Thx. My main problem was in using 'space-between'. Just updated task solution.